modules/core/src/config.ts

```diff
@@ -168,6 +188,20 @@ const formattedEosTokens = coins.reduce((acc: EosTokenConfig[], coin) => {
   return acc;
 }, []);
 
+const formattedFiatTokens = coins.reduce((acc: FiatTokenConfig[], coin) => {
+  if (coin instanceof FiatCoin) {
+    acc.push({
+      name: coin.fullName,
+      type: coin.name,
+      coin: coin.network.type === NetworkType.MAINNET ? 'fiat' : 'tfiat',
+      network: coin.network.type === NetworkType.MAINNET ? 'Mainnet' : 'Testnet',
+      tokenContractAddress: '',
```

Comment:
why do we need a `tokenContractAddress` and why is it always just an empty string?

Reply:
Removed. We don't need the `tokenContractAddress` for Fiat. Thanks!